src/js/deon-main.js

```diff
@@ -1610,7 +1610,7 @@ function accessDownloadOrModal (e, el) {
 }
 
 function canDownload () {
-  return hasGoldAccess() || (session.user && session.user.type && session.user.type.indexOf('artist') > -1)
+  return session.user && session.user && session.user.hasDownload
```

Comment:
Unnecessarely checking twice for `session.user`
```suggestion
  return session.user && session.user.hasDownload
```